awswrangler/timestream.py

```diff
@@ -240,21 +249,22 @@ def write(
     _logger.debug("cols_names: %s", cols_names)
     batches: List[List[Any]] = _utils.chunkify(lst=_df2list(df=df[cols_names]), max_length=100)
```

Comment:
Have you tried this PR code on a very large dataframe? Asking because I would expect this line to be a huge bottleneck. It effectively accesses every row in the dataframe in a non-distributed way...

We might need to refactor the code to leverage [apply](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.apply.html#pandas.DataFrame.apply) instead